.metadata/.log

```diff
@@ -0,0 +1,44 @@
+!SESSION 2016-02-24 18:58:15.287 -----------------------------------------------
+eclipse.buildId=4.4.2.M20150204-1700
```

Comment:
files like this should NOT be checked in. please add the file extension to the .gitignore file